scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -22,7 +22,7 @@
 # (c) 2016-2019 solidity contributors.
 #------------------------------------------------------------------------------
 FROM buildpack-deps:focal AS base
-LABEL version="4"
+LABEL version="5"
```

Comment:
Does the version need to be revised up? iiuc, there are no z3 related changes in this docker file.

Reply:
I think we need to change it to force the image to be rebuilt with the new z3? It installs static z3 via the PPA

Reply:
It installs the latest Z3 package from the PPA, which now contain Z3 4.8.10. So the newly built image will be different. Bumping version has the nice side-effect of invalidating Docker cache and ensuring that the image will really be rebuilt.